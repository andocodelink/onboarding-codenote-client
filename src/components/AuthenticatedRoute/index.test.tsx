import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { shallow, mount, render } from 'enzyme';
import { Button } from 'react-bootstrap';

import { AuthenticatedRoute, mapStateToProps } from './index';

describe('Home screen', () => {
  const userHasAuthenticated = () => ({});
  const jdomAlert = window.alert;

  beforeAll(() => {
    window.alert = jest.fn();
  });

  afterAll(() => {
    window.alert = jdomAlert;
  })
  
  test('renders', () => {
    const wrapper = render(<MemoryRouter><AuthenticatedRoute id='id1' component={Button as any} props={{id: 'childId1'}}/></MemoryRouter>);
    expect(wrapper).toMatchSnapshot();
  });

})