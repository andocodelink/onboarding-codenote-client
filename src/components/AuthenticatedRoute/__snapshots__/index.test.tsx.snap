// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home screen renders 1`] = `null`;
